src/NUnitFramework/framework/Constraints/DelayedConstraint.cs

```diff
@@ -35,9 +34,159 @@ namespace NUnit.Framework.Constraints
     ///</summary>
     public class DelayedConstraint : PrefixConstraint
     {
+        /// <summary>
+        /// Allows only changing the time dimension of the polling interval of a DelayedConstraint
+        /// </summary>
+        public class WithRawPollingInterval : DelayedConstraint
```

Comment:
Another nit pick: I got confused because the nested classes are listed in the reverse order in which the syntax appears normally.